metricsbp/baseplate_hooks.go

```diff
@@ -10,7 +12,7 @@ import (
 
 const (
 	success = "success"
-	fail    = "fail"
+	failure = "failure"
```

Comment:
For easier transition, can we keep both `fail` and `failure` for at a while (e.g. two weeks), then remove the wrong one (`fail`)?

Reply:
good call